conan/tools/cmake/toolchain.py

```diff
@@ -515,6 +515,12 @@ def _get_toolset(self, generator):
                     return "version=14.{}".format(minor)
                 else:
                     return "v14{}".format(minor)
+        elif compiler == "clang":
+            if generator and "Visual" in generator:
```

Comment:
Why cannot we use other generators such as `cmake` ?

Reply:
This is the CMake generator. You need to select one of the CMake supported ones: ``MinGW Makefiles``, ``Visual Studio XX YYYY``, etc.

Also, ``CMakeToolchain`` will only support the ``CMakeDeps`` generator (based on find_package()), but not the legacy ``cmake``, ``cmake_find_package[_multi]``, becase these later ones will be removed in 2.0.

Reply:
Ok, I got it, thank you.